### DISCUSSION TITLE
About Nokia phone bootloader unlock state

### DISCUSSION THREAD

**Discussion by HikariCalyx** [zenfyrdev/bootloader-unlock-wall-of-shame#6] [General]
1. Most of MediaTek based Nokia Android phones can utilize [MTKClient](https://github.com/bkerler/mtkclient) to perform bootloader unlock.
2. Nokia Android phones released until early 2019 can request unofficial bootloader unlock from https://hikaricalyx.com/request-bootloader-unlock - although online access is still required, it's not connected to HMD's official server, but [our own API server](https://api.hikaricalyx.com/).
3. As for prototype ABL images with bootloader unlock capability, there are 3 of them that could be used 100% offline: 7.2, 8.3 and 5.3. All other prototype ABL images may require using an internal flash tool named HDK to achieve bootloader unlock. You can find all prototype ABL images we have collected on https://fih-firmware.hikaricalyx.com/protoabl .
4. Avoid buying unisoc-based or USA exclusive Nokia phones for whatever reason.

**Comment by zenfyrdev** [comment]
Hi, thanks for the info. I've updated the page accordingly!
